dask/dataframe/tests/test_io.py

```diff
@@ -630,16 +633,20 @@ def test_to_castra():
     finally:
         c.drop()
 
-    dsk, keys = a.to_castra(compute=False)
-    assert isinstance(dsk, dict)
-    assert isinstance(keys, list)
-    c, last = keys
-    assert last[1] == a.npartitions - 1
+    delayed = a.to_castra(compute=False)
+    assert isinstance(delayed.dask, dict)
```

Comment:
This would be better to check if it was an instance of `Delayed`. All `.dask` attributes are dicts.